config/locales/en.yml

```diff
@@ -110,6 +111,7 @@ en:
       address_line_4: "Address 4"
       postcode: "Postcode"
       date_of_birth: "Date of Birth"
+      gender: "Gender"
```

Comment:
Can this be "Payroll Gender" instead. I'm wary of support agents, or claims checkers using this value to assume pronouns for users.

Reply:
that's a good shout 👍